Logger/FileLogger.cs

```diff
@@ -0,0 +1,24 @@
+﻿
+
+namespace Logger;
+
+public class FileLogger : BaseLogger
+{
+    private readonly string filePath;
+    private string? _LogText;
+    public string? LogText => _LogText;
+    public FileLogger(string className, string path)
+    {
+        ClassName = className;
+        filePath = path;
+    }
+    public override void Log(LogLevel logLevel, string message)
+    {
+        StreamWriter sw = File.AppendText(filePath);
+        _LogText = $"{DateTime.Now} {nameof(FileLogger)} {logLevel} {message}";
```

Comment:
```suggestion
        _LogText = $"{DateTime.Now} {ClassName} {logLevel} {message}";
```
`nameof` should be used when setting the classname.